src/sail_lean_backend/Sail/Sail.lean

```diff
@@ -249,6 +249,10 @@ def sail_barrier (_ : α) : PreSailM RegisterType c ue Unit := pure ()
 
 end ConcurrencyInterface
 
+def print (_ : String) : Unit := ()
+
+def print_endline (_ : String) : Unit := ()
+
```

Comment:
Why do we have these functions here? they should be effectful.

If that's for Riscv, they are defined in the model itself and they are defined in the RisccExtra.lean file in my branch of the riscv model https://github.com/ineol/sail-riscv/tree/lean-options

Reply:
Ah, thanks! Will delete.

Reply:
Do you plan on PR'ing that branch soon?